<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LbpTopSettings</class>
 <widget class="QDialog" name="LbpTopSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LBP-TOP参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="emotion_ai_main_window.qrc">
    <normaloff>:/EmotionAI/Resources/app1.png</normaloff>:/EmotionAI/Resources/app1.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>301</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
</string>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>邻域(默认值)</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="PxyLabel">
         <property name="text">
          <string>Pxy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="PxySpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="PxtLabel">
         <property name="text">
          <string>Pxt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="PxtSpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="PytLabel">
         <property name="text">
          <string>Pyt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="PytSpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>半径</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="RxLabel">
         <property name="text">
          <string>Rx</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="RxSpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="RyLabel">
         <property name="text">
          <string>Ry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="RySpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="RtLabel">
         <property name="text">
          <string>Rt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="RtSpinBox">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="emotion_ai_main_window.qrc"/>
 </resources>
 <connections/>
</ui>
